cmd/server/config.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/kubernetes-incubator/cri-o/server"
+	"github.com/opencontainers/runc/libcontainer/selinux"
+	"github.com/urfave/cli"
+)
+
+const (
+	ocidRoot   = "/var/lib/ocid"
+	conmonPath = "/usr/libexec/ocid/conmon"
+	pausePath  = "/usr/libexec/ocid/pause"
+)
+
+var commentedConfigTemplate = template.Must(template.New("config").Parse(`
+# Commented version of ocid configuration file. Generated automatically.
+
+# The "ocid" table contains all of the server options. There are no
+# bare options. "ocid" contains global options that don't really
+# relate to any particular sub-table.
+[ocid]
+
+# root is a path to the "root directory" where all information not
+# explicitly handled by other options will be stored.
+root = "{{ .Root }}"
+
+# sandbox_dir is the directory where ocid will store all of its sandbox
+# state and other information.
+sandbox_dir = "{{ .SandboxDir }}"
+
+# container_dir is the directory where ocid will store all of its
+# container state and other information.
+container_dir = "{{ .ContainerDir }}"
+
+# The "ocid.api" table contains settings for the kubelet/gRPC
+# interface (which is also used by ocic).
+[ocid.api]
+
+# listen is the path to the AF_LOCAL socket on which cri-o will listen.
+# This may support proto://addr formats later, but currently this is just
```

Comment:
We should only cover what is there now.  Remember an Admin will use this.  We can add comments like this in code, but not in the config.